modules/oe_content_organisation/oe_content_organisation.post_update.php

```diff
@@ -69,3 +69,28 @@ function oe_content_organisation_post_update_00005(): void {
   $field_config->set('label', 'Contacts');
   $field_config->save();
 }
+
+/**
+ * Creates the subject field.
+ */
+function oe_content_organisation_post_update_00006(): void {
+  // Create the subject field.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_organisation') . '/config/post_updates/00006_subject_field');
```

Comment:
We started to prefix the numbers with the module version like 20006, we should start doing it here too.